@here/harp-mapview/lib/geometry/TileGeometryCreator.ts

```diff
@@ -222,12 +226,30 @@ export class TileGeometryCreator {
             }
             return filter(technique);
         };
+        this.createTextElements(tile, decodedTile, textFilter);
 
-        this.preparePois(tile, decodedTile);
+        this.createLabelRejectionElements(tile, decodedTile);
+    }
```

Comment:
what happened to preparePois?

Reply:
line 215, this was also a refactoring, I moved textFilter to just above where it is needed.

I try to avoid doing this, but in this case, it made sense.